pkg/detector/detector.go

```diff
@@ -63,6 +63,22 @@ import (
 	"github.com/karmada-io/karmada/pkg/util/restmapper"
 )
 
+var (
+	propagationPolicyMarkedLabels = []string{
+		policyv1alpha1.PropagationPolicyPermanentIDLabel,
```

Comment:
Now we should also need to clean up the namespace and name label of the PropagationPolicy. Otherwise, it will result in residue.

Same as ClusterPropagationPolicy.

Reply:
Thanks for your remainder.